vespalib/src/vespa/vespalib/util/cgroup_resource_limits.h

```diff
@@ -0,0 +1,47 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <cstdint>
+#include <functional>
+#include <optional>
+#include <string>
+#include <map>
+
+namespace vespalib {
+
+/*
+ * Class for getting cgroup resource limits.
```

Comment:
Please add some details on which cgroup setups that are supported.

Reply:
Done.